lib/config/fileconf_test.go

```diff
@@ -332,6 +334,34 @@ func TestAuthenticationSection(t *testing.T) {
 	}
 }
 
+func TestAuthenticationConfig_Parse_StaticToken(t *testing.T) {
+	roles := []string{"Auth", "Node", "Proxy"}
+	joinedRoles := strings.Join(roles[:], ",")
+	var token StaticToken = StaticToken(fmt.Sprintf("%s:token", joinedRoles))
```

Comment:
No brainer simplifications:

```suggestion
	roles := []string{"Auth", "Node", "Proxy"}
	joinedRoles := strings.Join(roles, ",")
	token := StaticToken(fmt.Sprintf("%s:token", joinedRoles))
```

but I'd much rather do:

```suggestion
	token := StaticToken("Auth,Node,Proxy:token")
```

much easier to read.

Reply:
Simplified the test to be table driven, but for convenience I still do the string concatenation in there. I think this is resolved